cg/cli/upload/base.py

```diff
@@ -66,8 +67,10 @@ def upload(context: click.Context, family_id: Optional[str], restart: bool):
             return
 
         # Update the upload API based on the data analysis type (MIP-DNA by default)
-        if Pipeline.BALSAMIC in case_obj.data_analysis:
+        if case_obj.data_analysis == Pipeline.BALSAMIC:
```

Comment:
We'll stop uploading Balsamic UMI and Balsmaic QC cases if you change this. 

```
class Pipeline(StrEnum):
    BALSAMIC: str = "balsamic"
    BALSAMIC_QC: str = "balsamic-qc"
    BALSAMIC_UMI: str = "balsamic-umi"
```

```suggestion
        if Pipeline.BALSAMIC in case_obj.data_analysis:
```

Reply:
good catch! yes

Reply:
Ah! Now I see why it was phrased like that 🤯